projects/components/src/lib/result-table/result-table.component.html

```diff
@@ -17,7 +17,7 @@
             </ng-container>
         </tbody>
     </table>
-    <div class="table-pagination" *ngIf="state === 'succeeded' && model && model.results.length > 0 && model.totalResults > model.results.length">
```

Comment:
so a page with { results: [1, 2, 3], totalResults: 3, pageSize: 20 } would still not see pagination